src/feedback.tsx

```diff
@@ -299,13 +299,20 @@ export const FeedbackScreen: React.FC<FeedbackScreenProps> = ({
 export const SuccessFeedbackScreen: React.FC<AssetFeedbackProps> = ({dataAttributes, ...props}) => {
     const {isTabletOrSmaller} = useScreenSize();
     const {skinName} = useTheme();
+    const isFirstRender = useIsFirstRender();
```

Comment:
does it make sense to include  this logic inside the IconSuccess?

Reply:
mmm maybe? I added it this way as we already have a skipAnimation prop with other conditions in the stepper, also we have another icons that need this... but I think it make sense to disable this animations for those icons , I can make that changes